src/components/victory-label.jsx

```diff
@@ -194,20 +195,24 @@ export default class VictoryLabel extends React.Component {
 
   render() {
     this.getCalculatedValues(this.props);
+
+    const { fontSize, fontFamily, fill } = this.style;
+    const stroke = this.style.stroke !== "transparent" ? this.style.stroke : null;
+
     return (
-      <text x={this.props.x} y={this.props.y} dy={this.dy} dx={this.dx}
-        textAnchor={this.textAnchor}
-        transform={this.transform}
-        style={this.style}
+      <Text
+        x={this.props.x}
+        y={this.props.y}
+        dy={this.dy}
+        dx={this.dx}
+        alignment={this.textAnchor}
+        transform={new Transform().translate(this.transform)}
+        font={{ fontSize, fontFamily }}
+        stroke={stroke}
+        fill={fill}
       >
-        {this.content.map((line, i) => {
-          return (
-            <tspan key={i} x={this.props.x} dy={i ? this.lineHeight : undefined}>
```

Comment:
I think this wont matter since art is splitting text with newline characters anyway, and calculating it's own line height. https://github.com/sebmarkbage/art/blob/master/modes/canvas/text.js#L30